vendor/zkllvm-metacraft-circuits/src/utils/file_utils.h

```diff
@@ -0,0 +1,33 @@
+#include "boost/filesystem.hpp"
+using namespace boost::filesystem;
+
+namespace file_utils {
+    struct non_existing_path : public std::runtime_error {
+        template<typename T>
+        non_existing_path(T&& arg) : runtime_error(std::forward<T>(arg)) {
+        }
```

Comment:
Rename this to NonExistentPath since I've seen you use PascalCase for naming structs